pkg/applicationserver/io/packages/alcsync/v1/factory.go

```diff
@@ -0,0 +1,104 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alcsyncv1
+
+import (
+	"encoding/binary"
+	"time"
+
+	"go.thethings.network/lorawan-stack/v3/pkg/gpstime"
+	"go.thethings.network/lorawan-stack/v3/pkg/ttnpb"
+	lorautil "go.thethings.network/lorawan-stack/v3/pkg/util/lora"
+)
+
+// CommandFactory is the factory for commands.
```

Comment:
In general in Go we don't really use this 'factory' notation. I would remove `CommandFactory` and just leave the functions at the top level of the package.

Reply:
Should I leave the functions in the `factory.go` file or move them to the `alcsync.go` file altogether?

Reply:
I am ok with moving it to `alcsync.go`.